### DISCUSSION TITLE
How to set /NSM log_size_limit

### DISCUSSION THREAD

**Discussion by glwallum** [Security-Onion-Solutions/securityonion#4144] [Unsupported Versions]
Hello,

My sensor disk keeps filling up. I found that the /nsm drive was full

I am trying to limit the disk % in /opt/so/saltstack/local/pillar/minions/(node).sls, but can't find the syntax on how to set it

Is there documentation for this anywhere?

**Comment by gcsrellis** [comment]
[#3738](https://github.com/Security-Onion-Solutions/securityonion/discussions/3738)  Is probably what you are experiencing.  Updating to 2.3.50 should also fix the issue.

**Reply by glwallum** [reply]
I am on 2.3.50. The issue is that the /dev/mapper/system-nsm drive is filling to 100% when it is supposed to start purging at 90%

**Comment by InfosecGoon** [comment]
Is the disk filling with pcaps? It looks like the setting for that is in /opt/so/conf/steno/config -- DiskFreePercentage is the threshold to start deleting old records. Is yours set to 10 on the sensor box?

**Reply by glwallum** [reply]
Yes, it is set to 10.

**Reply by InfosecGoon** [reply]
Any interesting errors in /opt/so/log/stenographer?

**Reply by glwallum** [reply]
I ended up reinstalling our sensor and it appears to be functioning correctly now. Thank you for the troubleshooting steps

**Comment by glwallum** [comment]
Reinstalled sensor
